src/functionalTest/resources/request/casedata/ccd-callback-casedata-caseworker-submit-test-case.json

```diff
@@ -0,0 +1,5 @@
+{
+  "data": {
+
```

Comment:
We might be able to simply have {} in the request. You may even be able to pass empty JSON in the Java code to avoid the need for this redundant file